webtool/test_api.py

```
@@ -2622,3 +2622,250 @@ def test_export_project_zip_unknown_project_404(client):
+
+    `rename_file` gibt Ziele OHNE `.weg` herein. Ist der alte Platz beim Rücklauf wieder belegt,
+    wird nicht zurückbenannt (sonst überbügelte man den fremden Schreibvorgang) — die schon
+    umbenannte Datei bliebe dann aber unter dem NEUEN Namen in jeder Auflistung stehen, also
+    genau der Zustand, den `_umbenennen_oder_keines` ausschliesst. Sie gehört in den
+    unsichtbaren Namensraum, und der Fall gehört ins Protokoll (Bot-Befund an #460).
+    """
+    tdir = tmp_path / "Demo" / "transkripte"
+    (tdir / "S1.edit.json").write_text('{"alt": true}', encoding="utf-8")
+    echt_rename = os.rename
+
+    def fake(src, dst):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `fake`

Add return type annotation: `None`

(ANN202)